### PR TITLE
fix: allowing bottom tabs to be selectable as text

### LINKED ISSUE #32755: [Bug]: error log is not copy-able
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

when I have an erroneous query, I see the error both in response and error tabs. In neither of these tabs can I copy the error itself - this is important because I may want to search on google or stackoverflow etc. to rectify the error. 

This was working before, so treating this as a regression.

https://github.com/appsmithorg/appsmith/assets/94514895/2ed2d611-4161-4843-bb32-927b75e37860

### Steps To Reproduce

1. create an erroneous query
2. execute it
3. see that there's an error response
4. try to copy the error --> not able to copy

### Public Sample App

_No response_

### Environment

Production

### Severity

High (Blocker to building or releasing)

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by AmanAgarwal041**
## Description
Enabling the bottom tabs overall for users to select the test for errors, response, logs etc. The tailwind css class `select-none` which was added to the whole of the application was getting applied to this area as well by inheriting the properties. Added `select-text` property which enables the user to select anything inside the container as text.

Fixes #32755
_or_  
Fixes `Issue URL`
> [!WARNING]  
> _If no issue exists, please create an issue first, and check with the maintainers if the issue is valid._

## Automation

/ok-to-test tags="@tag.IDE"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9065702943>
> Commit: 9aeb5f4b884f129f6f8f0752f484e191aeae1f3a
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9065702943&attempt=1" target="_blank">Click here!</a>

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by ayushpahwa — COMMENTED** [review]
Changes look good, can we explore if any tests can be added here?

File: app/client/cypress/tags.js [hunk 1]
```diff
@@ -68,5 +68,6 @@ module.exports = {
     "@tag.Visual",
     "@tag.Module",
     "@tag.Workflows",
+    "@tag.Debugger",
```
Comment (sneha122) [inline app/client/cypress/tags.js:71]: @AmanAgarwal041 Is this tag needed? I dont see any cypress test cases but tag is there, if not needed, can we please remove it?

Comment (AmanAgarwal041) [reply app/client/cypress/tags.js:71]: This tag was not included but there are test cases related to debugger tabs that use this tag. So I included it in tag.js to be able to run the @tag.Datasource test cases.

**Comment by sneha122** [comment]
@AmanAgarwal041 Tested the changes, looks good, noticed one thing though, in error tabs the cursor does not change to selection one, it stays as default. Please check

https://github.com/appsmithorg/appsmith/assets/30018882/5035c355-8ce6-4968-a09f-41f1a4329139

**Comment by AmanAgarwal041** [comment]
> @AmanAgarwal041 Tested the changes, looks good, noticed one thing though, in error tabs the cursor does not change to selection one, it stays as default. Please check
> 
>  Screen.Recording.2024-05-10.at.12.52.07.PM.mov

Multiple components collapsible, json viewer etc inside this have `cursor: default` so I think its better to leave this till the text is getting selected. @sneha122

File: app/client/cypress/tags.js (hunk 1 above)
Comment (hetunandu) [reply app/client/cypress/tags.js:71]: We recently removed the Debugger tag and moved it into @tag.IDE. Please use the IDE for any debugger specific tests

**Review by sneha122 — APPROVED** [review]

**Pull request merged**